functional_tests/mp/06-5-share-child-non-service.yea

```diff
@@ -2,8 +2,7 @@ id: 0.mp.06-5-share-child-non-service
 plugin:
   - wandb
 tag:
-  skips:
```

Comment:
I wanted to have conditional skip, but we setup the env after we read the scheme so for now it's hard-coded